.circleci/config.yml

```diff
@@ -4,62 +4,97 @@
 #
 version: 2
 jobs:
-  build:
+  mysql:
     docker:
       # specify the version you desire here
-      - image: circleci/ruby:2.5.1-node-browsers
-
-      # Specify service dependencies here if necessary
-      # CircleCI maintains a library of pre-built images
-      # documented at https://circleci.com/docs/2.0/circleci-images/
+      - image: circleci/ruby:2.5-node-browsers
       - image: circleci/mysql:5.7
-      - image: circleci/postgres:9.4
-
-    working_directory: ~/repo
-
+    working_directory: &workdir ~/repo
     steps:
       - checkout
-
-      # Download and cache dependencies
+      # Restore bundle cache
       - restore_cache:
           keys:
-          - v1-dependencies-{{ checksum "Gemfile.lock" }}
-          # fallback to using the latest cache if no exact match is found
-          - v1-dependencies-
-
+            - v1-dependencies-{{ .Branch }}
+            - v1-dependencies-
+      - run:
+          name: Bundle Install
+          command:
+            bundle install --path=vendor/bundle
+      # Store bundle cache
+      - save_cache:
+          key: v1-dependencies-{{ .Branch }}-{{ checksum "Gemfile.lock" }}
+          paths:
+            - ./vendor/bundle
+      - run: sudo apt install -y postgresql-client
       - run:
-          name: install dependencies
+          name: Install 'phantomjs'
```

Comment:
`phantomjs` was unmaintained and EOL the last time I checked, so I don't think we want to rely on it for testing.

Reply:
removed